src/types/markers.js

```diff
@@ -732,6 +732,11 @@ export type NoPayloadUserData = {|
   innerWindowID?: number,
 |};
 
+export type UrlMarkerPayload = {|
```

Comment:
Is this type only for test or do we have a marker type like this?
Edit: oh nvm, it looks the Andrew's patch will add this in mozilla-central.